updates.txt

```diff
@@ -0,0 +1,9 @@
+Updates 16Dec2021: Previously, the pet object was empty. Now it contains a name, an age and the growUp function, which ages the pet 1 year when called. A few tests have also been written, asking whether the pet is an object, and whether that object has a name and age and whether the growUp function actually increases the age.
```

Comment:
This is some good information, adding a changelog like this is a common practice and to be encouraged! Usually this is kept in a `CHANGELOG.md` file, which contains a series of short statements describing what has changed. There is a whole convention around doing this which would be too involved to include here, but a quick example would be something like:
```
##(2021-12-16)
###ADDED
- Name, age and growUp function added to Pet
- Tests added
```
Refer to the markdown doc I mentioned in the README  comments for more info